src/components/NcInputField/NcInputField.vue

```diff
@@ -465,16 +467,19 @@ export default {
 
 	&__input:focus + &__label,
 	&__input:not(:placeholder-shown) + &__label {
-		inset-block-start: -6px;
+		inset-block-start: -8px;
 		font-size: 13px; // minimum allowed font size for accessibility
+		font-weight: 500;
+		border-radius: 9px 9px 0 0; // height / 2 for pill style
```

Comment:
```suggestion
		border-radius: var(--default-grid-baseline) var(--default-grid-baseline) 0 0;
```

Reply:
I am fine with both, pill style was the conclusion after the last design review call.